Sming/Components/Storage/Tools/hwconfig/config.py

```diff
@@ -5,7 +5,13 @@
 import os, partition, storage
 from common import *
 from builtins import classmethod
-from rjsmin import jsmin
+
+try:
+    from rjsmin import jsmin
+except ModuleNotFoundError as err:
```

Comment:
@mikee47 Can you replace `ModuleNotFoundError` with `ImportError`. It is also compatible with older python versions.